jetson/nav/stateMachine.cpp

```diff
@@ -270,10 +270,6 @@ NavState StateMachine::executeOff()
         mCompletedWaypoints = 0;
         mTotalWaypoints = mRover->roverStatus().course().num_waypoints;
 
-        if( !mTotalWaypoints )
-        {
-            return NavState::Done;
-        }
         return NavState::Turn;
```

Comment:
why does execute off return turn?

Reply:
This part of `executeOff` is checking if the base station sends us the `is_auton` boolean, so within that branch it'll check if it has to move or not. The comment above should explain it, but basically it should have a transition from off to the first move the rover does (which is turn to face a waypoint)

Reply:
ok...some of these variables/functions definitely could be renamed